src/common/pa_converters.c

```diff
@@ -329,24 +352,124 @@ static const double const_1_div_2147483648_ = 1.0 / 2147483648.0; /* 32 bit mult
 
 /* -------------------------------------------------------------------------- */
 
+static inline int setTruncRoundingModePriv()
```

Comment:
I assume "trunc" is short for "truncation". Please use full words.

Also what is "Priv"? Is that Privilege or Private or something else?
Also does "Priv" have something to do with rounding toward zero.

Maybe "SetTruncationRoundingModeTowardZero()"?